tests/filesystem.go

```diff
@@ -224,7 +224,7 @@ func setupDisk(ctx context.Context, disk *types.Disk, diskIndex int, imageSize i
 	loopdev := disk.Device
 	defer func() {
 		if err != nil {
-			destroyDevice(loopdev)
+			_ = destroyDevice(loopdev)
```

Comment:
All of the changes in the `tests` package (and it's sub-packages) don't seem related to logging and should probably be moved to their own commit.